pyanaconda/modules/localization/live_keyboard.py

```diff
@@ -81,8 +82,8 @@ def _convert_to_xkb_format(self, sources):
         # convert input "[('xkb', 'us'), ('xkb', 'cz+qwerty')]\n"
         # to a python list of '["us", "cz (qwerty)"]'
         try:
-            sources = eval(sources.rstrip())
-        except SyntaxError:
+            sources = ast.literal_eval(sources.rstrip())
+        except (SyntaxError, ValueError):
```

Comment:
Seems to need also TypeError here - https://docs.python.org/3/library/ast.html#ast.literal_eval